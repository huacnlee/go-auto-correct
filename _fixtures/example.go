package foo

import "fmt"

const (
	// 这是comment单行
	foo = "这是一个Go的提示信息" // 测试after注释
	/* 测试multi-line字符串 */
	bar = `这是Go长字符串
	这是string第二行`
)

func main() {
	fmt.Println("准备打印Foo的结果:", foo)
}
