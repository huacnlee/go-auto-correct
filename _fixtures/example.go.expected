package foo

import "fmt"

const (
	// 这是 comment 单行
	foo = "这是一个 Go 的提示信息" // 测试 after 注释
	/* 测试 multi-line 字符串 */
	bar = `这是 Go 长字符串
	这是 string 第二行`
)

func main() {
	fmt.Println("准备打印 Foo 的结果:", foo)
}
